src/libguac/client.c

```diff
@@ -633,6 +668,40 @@ static void* __webp_support_callback(guac_user* user, void* data) {
 }
 #endif
 
+/**
+ * A callback function which is invoked by guac_client_owner_supports_required()
+ * to determine if the owner of a client supports the "required" instruction,
+ * updating the flag to indicate support.
+ * 
+ * @param user
+ *     The guac_user that will be checked for "required" instruction support.
+ * 
+ * @param data
+ *     A pointer to an int containing the status for support of the "required"
+ *     instruction.  This will be 0 if the owner does not support this
+ *     instruction, or 1 if the owner does support it.
+ * 
+ * @return
+ *     Always NULL.
+ */
+static void* guac_owner_supports_required_callback(guac_user* user, void* data) {
+    
+    data = (void *) ((intptr_t) guac_user_supports_required(user));
```

Comment:
Same here - beware that `user` can be `NULL` if the owner has left the connection.

Reply:
I think this occurrence is okay, as there is no de-reference of a pointer within this function, and the called function `guac_user_supports_required()` does check to make sure `user` is not `NULL`.

Or would you prefer to handle the case of `user` being `NULL` within this function, as well?

Reply:
Ah, OK. If `guac_user_supports_required()` already handles this case, I have no issue with this.